x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/export.cy.ts

```diff
@@ -19,12 +19,11 @@ import { TOASTER } from '../../../screens/alerts_detection_rules';
 import { TIMELINE_CHECKBOX } from '../../../screens/timelines';
 import { createTimeline } from '../../../tasks/api_calls/timelines';
 import { expectedExportedTimeline, getTimeline } from '../../../objects/timeline';
-import { deleteTimelines } from '../../../tasks/common';
 
 // FLAKY: https://github.com/elastic/kibana/issues/165744
 describe('Export timelines', { tags: ['@ess', '@serverless'] }, () => {
   before(() => {
-    deleteTimelines();
+    login();
```

Comment:
Now this one is interesting. Can you elaborate why we need to login now and before we didn't need to?

Reply:
I might messed up when solving conflicts :D

Reply:
For what I see we were doing a login on beforeEach, but just with the before hook is enough, we don't need to login twice :)